docs/reporting/overview.md

```diff
@@ -0,0 +1,56 @@
+# Overview
+
+Reporting API allows you to query all transactions taking place at any of your MobilePay payment locations (payment points).
+
+The main objective is to make it easier for you to do reconciliation on your side. We at MobilePay will provide details of transactions to help you with that, with the products that MobilePay offers. This will be done by summarizing payment, fees, refunds. These are summarized in further detail below.
```

Comment:
**"... with the products that MobilePay offers"** - this part of sentence feels weird, I'd remove it. If you decide to leave it, then I think "the" is not needed before "products".

**"This will be done by summarizing payment, fees, refunds. These are summarized in further detail below."** - not sure I understand what this sentence says. "payment" should be plural.

Reply:
I changed last 3 sentences to this:

> We at MobilePay will provide details of transactions to help you with that. This will be done by summarizing payment, fees and refunds into single list of transactions.